test/regression-wasm/test-locals-spec.kore

```diff
@@ -1,17 +1,17 @@
-[org'Stop'kframework'Stop'attributes'Stop'Source{}("Source(wasm-semantics/tests/proofs/locals-spec.k)")]
+[org'Stop'kframework'Stop'attributes'Stop'Source{}("Source(/Users/emarzion/kore/wasm-semantics/tests/proofs/locals-spec.k)")]
```

Comment:
I'm not sure if this matters, but this doesn't look intentional (all 4 changes to this file are the same as this one).

Reply:
I've had this issue before, and Anna said it was not an issue since it would be corrected once the tests were regenerated.  If we wanted to correct this I believe that `trim-source-paths.sh` would again need to be modified (`/home/jenkins-slave/workspace/` replaced with the appropriate directory I think).  Any thoughts @ana-pantilie ?

Reply:
In the meantime I think all these issues should be resolved now.